src/main/groovy/jenkins/automation/builders/BddSecurityJobBuilder.groovy

```diff
@@ -68,28 +68,29 @@ class BddSecurityJobBuilder {
                         sed -i 's/<baseUrl><\\/baseUrl>/<baseUrl>${baseUrl}<\\/baseUrl>/g' config.xml
 
                         sed -i 's/<defaultDriver path.*/<defaultDriver path="${chromedriverPath}">Chrome<\\/defaultDriver>/g' config.xml
-
-                        ant resolve
-
-                        ant jbehave.run""")
+                        
+                        ./gradlew cleanTest test""")
             }
         }
 
         baseJob.with {
-            /**
-             *  file path pattern of the JBehave reports
-             */
+        
+            configure { project ->
+                project / publishers / 'com.github.bogdanlivadariu.jenkins.reporting.cucumber.CucumberTestReportPublisher' {
+                    'fileIncludePattern'('build/reports/cucumber/all_tests.json')
+                }
+            }
+        
             configure { project ->
-                project / publishers / 'xunit' / 'types' / 'JBehavePluginType' {
-                    'pattern'('reports/latest/*.xml')
+                project / publishers / 'net.masterthought.jenkins.CucumberReportPublisher' {
+                    'fileIncludePattern'('build/reports/cucumber/all_tests.json')
                 }
             }
-            /**
-             *  If the Total number of failed tests exceeds this threshold then fail the build
-             */
+            
             configure { project ->
-                project / publishers / 'xunit' / 'thresholds' / 'org.jenkinsci.plugins.xunit.threshold.FailedThreshold' {
-                    'failureThreshold'('0')
+                project / publishers / 'hudson.tasks.junit.JUnitResultArchiver' {
```

Comment:
So folks know why this is important: the JunitResultArchiver puts the test results in the place where you expect to see test results in Jenkins. In addition, it enables these tests to show up in the BuildFlow Test Aggregation (via build flow test aggregator plugin).